### ISSUE TITLE
[Todoist] menu bar erroring

### ISSUE THREAD

**Issue by dannybloomfield** [raycast/extensions#14409]
### Extension

https://www.raycast.com/doist/todoist

### Raycast Version

1.82.4

### macOS Version

15.0

#### Error:
```
Error: Worker terminated due to reaching memory limit: JS heap out of memory

[kOnExit]:worker:313:26
Worker.<computed>.onexit:worker:229:20
```

### Steps To Reproduce

todoist menu bar icon replaced with red ! triangle
clicked on it

### Current Behaviour

<img width="766" alt="image" src="https://github.com/user-attachments/assets/59cb4a67-b077-4767-ae9d-8b802508ff6b">

error as demonstrated in sceenshot
reloaded extension, same outcome

### Expected Behaviour

open todoist task list from menu bar

**Comment by neehhaa06** [comment]
Hello!! @dannybloomfield :)

fix: #14409
```
The issue occurs because the Todoist extension in Raycast is using more memory than allowed, causing a "JS heap out of memory" error. This can happen due to inefficient memory use or a possible memory leak. To fix it, you can clear the cache, update or reinstall the extension, restart Raycast or your system, and report the problem if it persists.
```

**Comment by thomaslombart** [comment]
Thanks for bringing this up. It appears that this is a duplicate of issue https://github.com/raycast/extensions/issues/10127, so I'll mark it as such.
